guide/english/bootstrap/index.md

```diff
@@ -76,10 +76,16 @@ Bootstrap has thorough documentation with many <a href='https://getbootstrap.com
 In addition, you can find both <a href='https://bootswatch.com/' target='_blank' rel='nofollow'>free</a> and <a href='https://themes.getbootstrap.com/' target='_blank' rel='nofollow'>paid</a>
 themes that build on the Bootstrap framework to provide a more customized and stylish look.
 
+>#### Adding custom color to your website:
+
```

Comment:
this will help you add a custom color to your website just describe it once and use it anywhere in your website